_posts/2021-04-13-upgrade-notification-for-ase.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "Upgrade Notifications for App Service"
+author_name: "Michimune Kohno and Sanghmitra Gite"
+toc: true
+toc_sticky: true
+tags:
+    - Upgrade notifications
+    - Azure Monitor
+    - App Service Environment
+    - Logic Apps
+---
+
+We are happy to announce that the upgrade notifications for App Service Environment is available. With an upgrade notification you will be able to receive email, SMS text, or take action before a platform upgrade actually starts.
+
+The Azure App Service is a PaaS solution offering in Azure which constantly updates the platform. One of the top requests from customers is to receive a notification before an upgrade operation happens. This feature has been rolled out for App Service Environments (ASEs) across regions.
+
+> Support for public App Service scale units will be coming soon.
+
+## Overview
+
+The upgrade notification for App Service is essentially an Azure Monitor event. This means that you can set up your email address and/or SMS phone number when a notification is generated. You can also set up a trigger for your custom Azure Function or Logic App, which allows you to automatically take action to your resources. For example, you can automatically divert all the traffic to your ASE in one region which will be upgraded to ASE in another region in order to avoid any potential impact. Then you can automatically change the traffic back to normal when an upgrade completes. Please refer to [Logic App sample for automatic traffic diversion for Azure App Service](https://github.com/Azure-Samples/azure-logic-app-traffic-update-samples) for more details.
+
+## Viewing upgrade notifications
+
+On the Azure portal, go to **Home** > **Monitor** > **Service Health** > **Planned maintenance**. Here you can see all active (Either upcoming or in-progress) notification for the selected subscriptions. To make it easy to find App Service upgrade events, click **Service** box, check all App Service types and uncheck everything else.
+
+To see past notifications, navigate to **Health history** and filter **Planned maintenance** from the Health Event Type box.
+
+![Health history]({{ site.baseurl }}/media/2021/04/upgrade-notification-health-history.png)
+
+## Setting up alerts
```

Comment:
The title of this section implies that this will show how to set up alerts, but it is actually assuming that I have created a logic app, step 12 says 'Select the subscription that you have created your Logic app during the previous part.', but there is no previous part that let's the user create a logic app. Are you referring to the doc link in the overview section?

Reply:
Thanks. I copy pasted this part from the sample logic app document and missed to change the part. Updated.